lib/__tests__/build-files.test.js

```diff
@@ -60,14 +61,22 @@ describe('Build files', () => {
     clearBuildFolder();
   });
 
+  test('Generated HTML for each Markdown resource', function() {
```

Comment:
I think we've added this. Double add due to merge conflict ??

Reply:
Copied assets from /docs/assets test as well

Reply:
Thanks, have fixed them 😓